### PR TITLE
return an array for babel config

### PR DESCRIPTION
**Pull request by sivteck**

### CODE REVIEW

**Comment by sharangj** [comment]
@sivteck Please raise this PR again if needed. Closing this for now.

**Comment by sivteck** [comment]
One of the dependencies of ahead  [0], expects Rule.use to be an array whereas quintype-node-build exports rules with the use property as an object. Webpack documentation also mentions Rule.use can either be an array or function [1], this change was made some time after the webpack 4 update. This PR makes getBabelConfig return an array to fix the build error in ahead.

[0] https://github.com/karify/external-svg-sprite-loader
[1] https://webpack.js.org/configuration/module/#ruleuse

**Comment by sivteck** [comment]
I am able to build `prabhatkhabar` and `the-cue` repos without errors after this change

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quintype/build",
-  "version": "3.10.0",
+  "version": "3.10.1-fix-babelconfig.2",
```
Comment (sharangj) [inline package.json]: Revert this change and we are good to go.

Comment (sivteck) [reply package.json]: done 👍
